tests/zfs-tests/tests/functional/fault/scrub_after_resilver.ksh

```diff
@@ -32,16 +32,15 @@
 log_assert "Testing the scrub after resilver zedlet"
 
 # Backup our zed.rc
-zedrc_backup="$(mktemp)"
-log_must cp $ZEDLET_DIR/zed.rc $zedrc_backup
+zedrc_backup=$(zed_rc_backup)
 
-# Enable ZED_SCRUB_AFTER_RESILVER
-eval "sed -i 's/\#ZED_SCRUB_AFTER_RESILVER/ZED_SCRUB_AFTER_RESILVER/g' $ZEDLET_DIR/zed.rc"
+# Enable ZED_SCRUB_AFTER_RESILVER in zed.rc
+zed_rc_set ZED_SCRUB_AFTER_RESILVER
```

Comment:
`zed_rc_set ZED_SCRUB_AFTER_RESILVER 1` should fix the issue found by the builders:

```
Test: /usr/share/zfs/zfs-tests/tests/functional/fault/scrub_after_resilver (run as root) [00:11] [FAIL]
03:18:29.63 ASSERTION: Testing the scrub after resilver zedlet
03:18:29.81 SUCCESS: zpool create -f testpool mirror loop0 loop1
03:18:29.98 SUCCESS: zfs create testpool/testfs
03:18:30.14 SUCCESS: zfs set mountpoint=/mnt/testdir testpool/testfs
03:18:30.21 SUCCESS: zpool offline -f testpool loop0
03:18:30.24 SUCCESS: mkfile 16M /mnt/testdir/file1
03:18:30.85 SUCCESS: zpool replace testpool loop0 loop2
03:18:40.97 
03:18:40.97 ERROR: wait_scrubbed testpool 10 exited 1
```

Then this LGTM, thanks!